src/main/java/Duke.java

```diff
@@ -21,12 +20,14 @@ public class Duke {
             " Bye. Hope to see you again soon!\n" + lineBreaker;
 
     public boolean shouldExit = false;
+
     public void setShouldExit () {
         this.shouldExit = true;
     }
-    public static void printList(int currentTask){
+
+    public static void printList(int currentTaskNum){

```

Comment:
You can try to make your methods look more uniform by leaving a space in between the (int currentTaskNum) and {

Reply:
Thank you! I'll change it!